readme.md

```diff
@@ -38,7 +38,7 @@ Alternatively, we can go fully automatic by passing flags `--preset --lint --jes
 - Monorepo structure is supported, but make sure you run the script from the app directory,
   not the monorepo root.
 
-When using `react-native-ci-cli`, you can provide additional flags to modify its default behavior.
+When using `setup-ci`, you can provide additional flags to modify its default behavior.
```

Comment:
So maybe let's replace all setup-ci with npx setup-ci here

Reply:
done in 44a91c68502ea53e19097a9dfb459cae156d5d6f